src/main/scala/org/clulab/habitus/utils/ContextExtractor.scala

```diff
@@ -96,7 +97,7 @@ trait ContextExtractor {
       val contextSentences = getSentIDsInSpan(mention, windowSize)
       val instances = getInstancesInContext(contextType, contextSentences, allMentions)
       if (instances.nonEmpty) {
-        return instances.groupBy(identity).map(i => i._1 -> i._2.length).max._1.toLowerCase()
+        return instances.groupBy(identity).map(i => i._1 -> i._2.length).maxBy(_._2)._1
```

Comment:
It doesn't matter, but if one wanted to get rid of this return, one might use recursion on the windowSize.

Reply:
I don't think I know how to do that :'(

Reply:
It's not important.  Some people have taken an oath never to use "return" and have thought about ways to avoid it.  If it changes some day, I'll tag you.